file-formats/ints/examples/rm_test_adt_976.ints.json

```diff
@@ -0,0 +1,28 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Creation of scenario via ADT",
+    "originalLanguage": "en"
+  },
+  "generalInformation": {
+    "scenarioType": "SAPGENAI",
+    "scenarioTechnology": "SIDEBYSIDE",
+    "isExtensible": true,
+    "state": "Published",
+    "islmVersion": 1.0
+  },
+  "classInformation": {},
```

Comment:
Is `classInformation` required information? If so, I would expect content within the example.

Reply:
As we discussed earlier, for 2502 we are keeping only few fields for ADT UI and many fields we are hidings and some need to be filled in Original system only. So, this class information, we are hiding as of now, but definitely it's important for us and we'll unhide it in further release. Actually, we got UX review comment to rename the section as General information. Then, we had internal discussion within our team and decided to divide the section similar to our Fiori UI.
<img width="839" alt="image" src="https://github.com/user-attachments/assets/17945749-945e-43ba-894b-d3478fdbc9c1">

Reply:
Ok, I see. But in this case field classInformation must not be a required structure as it is stated right now.

This is not critical issue. It can be fixed somewhen next week. But for sure you can also fix it now.

Reply:
Please, don't resolve a conversations. This makes it hard for us to see which questions we need to follow up or what we should double-check after changes. It is sufficient to add your answer.

Reply:
As I mentioned earlier, we're keeping all the fields in the interface and hiding many fields which are not part of 2502. So this class information is for next scope.

But I'm not sure why it's giving empty set in object source JSON. May be because this is a structure?
Actually, I've hidden other tables which are empty from the get_content method and they're not part of the object source JSON. Also, I'm not passing the class information in get_content.

May I know why only empty structure is displayed in the object source JSON and not empty tables?

Reply:
Sorry, I didn't manage to answer earlier. :(

I think the structure is rendered by default, because it is marked with the `$required` annotation.

https://github.com/SAP/abap-file-formats/blob/4be83fafe5cc66f2e963b0c2724780efd41c3971/file-formats/ints/type/zif_aff_ints_v1.intf.abap#L273-L274

If I understood your last comment, the `classInformation` must not be set in any INTS object. Therefore, you could/should remove the `$required` annotation.